gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1917,6 +1919,10 @@ protected void configureRestClientGeneration(Project project, SourceSet sourceSe
           {
             task.setEnableArgFile(true);
           }
+          if (isPropertyTrue(project, ENABLE_FLUENT_API))
```

Comment:
Do you plan to change it later? I thought this would be enabled by default: (fluentAPi constructed by default and users to decide whether to use it)

Reply:
Yes, when we do GA, we'll turn this on by default.